app/models/parted.rb

```diff
@@ -39,6 +39,7 @@ def merge_embedded_parts_errors
 
     if errors.delete(:parts) == ["is invalid"]
       parts_errors = parts.each_with_object({}) do |part, result|
+        part.errors.messages.each { |_k, v| v.delete("is invalid") }
```

Comment:
I think this is a mistake as it's removing "is invalid" validation from fields.

For example if you put in an invalid slug you'd not expect this error to be removed.

Reply:
While I'm happy to keep `is invalid` if you feel it needs to be there, I personally do not feel it adds anything or helps the user diagnose the issue.  If there are other reasons that I'm unaware of as to why you feel this error is useful, I'd be grateful if you could help me understand.

I would point out that this error will never be shown alone - it'll always be accompanied by a more meaningful and helpful message (e.g. `can't be blank`).  And having both `is invalid` and `can't be blank` does offer any further assistance to the user in my view as they can see it can't be blank, so do not also need to be told it's invalid.

Sorry if I'm missing something.

Reply:
Have removed this commit - thanks.